phoenix-core/src/main/java/org/apache/phoenix/coprocessor/BaseScannerRegionObserver.java

```diff
@@ -104,7 +104,6 @@
     public static final String PHYSICAL_DATA_TABLE_NAME = "_PhysicalDataTableName";
     public static final String EMPTY_COLUMN_FAMILY_NAME = "_EmptyCFName";
     public static final String EMPTY_COLUMN_QUALIFIER_NAME = "_EmptyCQName";
-    public static final String SCAN_LIMIT = "_ScanLimit";
```

Comment:
@gjacoby126, I have removed the option for multiple index row repair. By default, the code was handling single row repair. The implementation of the multi-row repair was very basic and not effective. In order to simplify failure handling I have removed this option. We need to come up with a better design for it if we feel that we need it.